profiles/slurm-conda/config.yaml

```diff
@@ -1,9 +1,14 @@
 jobscript: "../slurm-jobscript.sh"
 cluster: "../slurm-submit.py"
 cluster-status: "../slurm-status.py"
+cluster-sidecar: "../slurm-sidecar.py"
+cluster-cancel: "scancel"
+max-jobs-per-second: 1
+max-status-checks-per-second: 10
 snakefile: "../../workflow/Snakefile"
 cores: 256
 jobs: 256
+latency-wait: 60
```

Comment:
This is the default option for latency-wait for the new profile. Also max-jobs-per-second: 1 and max-status-checks-per-second: 10 were also the default values.

Reply:
< OK >

Reply:
Lets put latency to 5 and we see if this needs any change

Reply:
I removed it completely since the default latency-wait is 5 seconds. Locally seems to work fine.